py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,521 @@
+        (default '~/.config/py3status/google_calendar.client_secret')
+    force_lowercase: Sets whether to force all event output to lower case.
+        (default False)
+    format: The format for module output.
+        (default '{events}|\?color=event \u2687')
+    format_date: The format for date related format placeholders. May be any Python
+        strftime directives for dates.
+        (default '%a %d-%m')
+    format_event: The format for the first toggleable event output.
+        (default '[\?color=event {summary}][ ({location})][ {time_to}]')
+    format_event_full: The format for the second toggleable event output.
+        (default '[\?color=event {summary}]
+        [\?color=time  ({start_time} - {end_time}, {start_date})]')
+    format_separator: The string used to separate individual events.
+        (default '\?color=separator \|')
+    format_time: The format for time related format placeholders (except {time_to}).
+        May use any Python strftime directives for times.
+        (default '%I:%M %p')
+    format_time_left: The format for used for the {time_to} placeholder when when the time
+        displayed is the time left until an event in progress is over.
+        (default '\?color=time ([\?if=days {days}d ]
+        [\?if=hours {hours}h ][\?if=minutes {minutes}m]) left')
+    format_time_to: The format used for the {time_to} placeholder.
+        (default '\?color=time ([\?if=days {days}d ]
+        [\?if=hours {hours}h ][\?if=minutes {minutes}m])')
+    ignore_all_day_events: Sets whether to display all day events or not.
+        (default False)
+    num_events: The maximum number of events to display.
+        (default 3)
+    thresholds: Thresholds for events. The first entry is the color for event 1,
+        the second for event 2, and so on.
+        (default [])
+    time_to_max: Threshold (in minutes) for when to display the {time_to}
+        string; e.g. if time_to_max = 60, {time_to} will only be displayed
+        for events starting in 60 minutes or less.
+        (default 180)
+    warn_threshold: The number of minutes until an event starts before a warning is
+        displayed to notify the user; e.g. if warn_threshold = 30 and an event is
+        starting in 30 minutes or less, a notification will be displayed. disabled (= 0)
+        by default.
+        (default 0)
+    warn_timeout: The number of seconds before a warning should be displayed again.
+        (default 300)
+
+Format placeholders:
+    {events} All the events to display.
+
+Format_event & format_event_full placeholders:
+    {description} The description for the calendar event.
+    {end_date} The end date for the event.
+    {end_time} The end time for the event.
+    {location} The location for the event.
+    {start_date} The start date for the event.
+    {start_time} The start time for the event.
+    {summary} The summary (i.e. title) for the event.
+    {time_to} The time until the event starts (or until it is over
```

Comment:
This should be `{format_time_to}` as it's something that can be formatted and not part of event data. Maybe just `format_timer` if it's going to use current and until.

Reply:
Great idea! It's much better now. I'm going to do the same for format_event and format_event_full.